.gitignore

```diff
@@ -5,5 +5,8 @@ spellcheck
 .DS_Store
 settings.json
 parser_testing.txt
+test_result.txt
+spellcheck.dSYM/
```

Comment:
Let's remove all references to `dSYM` files. Nobody should be working directly on their macs - use the VM for everything.

Reply:
done